templates/default/typescript/www/images/.taco-ignore

```diff
@@ -1,3 +0,0 @@
-This is a dummy file with the sole purpose of allowing to commit its parent directory (which would otherwise be empty) to git source control.
-
-It is not intended to be part of the template and will be excluded from the archive process (when doing "gulp prepare-templates").
```

Comment:
Looks like images folder is no longer (otherwise) empty and will contain cordova.png. In that case, do you want to remove the ".taco-ignore" file type from the ignored file lists in utils\templateManager.ts?